src/components/layout/footer.js

```diff
@@ -11,13 +11,16 @@ const Footer = () => (
         <img src={logo} alt="The COVID Tracking Project" />
         <ul>
           <li>
-            <a href="https://github.com/COVID19Tracking">GitHub</a>
+            <a href="/contact">Contact us</a>
```

Comment:
I _think_ we would want this to be `Contact Us` (capital `U`), to align with the caps in the nav/footer

Reply:
I say just "Contact"